.github/workflows/lint.yml

```diff
@@ -1,6 +1,9 @@
 name: Client Lint and Format
 
-on: [push]
+on:
+  push:
+    paths:
+      - 'client/**'
```

Comment:
Only we add/change client code, the lint test will run. It is the same as server test.